protocols/syslog/syslog.h

```diff
@@ -27,20 +28,14 @@
 #include <avr/pgmspace.h>
 #include "protocols/uip/uip.h"
 
-#define MAX_DYNAMIC_SYSLOG_BUFFER 500
+#define MAX_DYNAMIC_SYSLOG_BUFFER 100
 
 uint8_t syslog_send_P(PGM_P message);
```

Comment:
Yes this is possible but the syslog_sendf functions need more ram because the resulting string length is unkown.